src/components/loop-animation.js

```diff
@@ -14,13 +14,12 @@ AFRAME.registerComponent("loop-animation", {
 
   init() {
     this.mixerEl = findAncestorWithComponent(this.el, "animation-mixer");
+    this.currentActions = [];
 
     if (!this.mixerEl) {
       console.warn("loop-animation component could not find an animation-mixer in its ancestors.");
       return;
     }
-
-    this.updateClip();
```

Comment:
This is actually not needed since `update` is always called after `init`, so its just doing extra work